prebid_cache_client/client.go

```diff
@@ -45,6 +45,10 @@ type Cacheable struct {
 	Data       json.RawMessage
```

Comment:
```suggestion
	Data       json.RawMessage    `json:"value"`
```

Reply:
I'm concerned about this change due to backwards compatibility. Thoughts?

Reply:
I think it's fine. The cache client builds the json piece by piece into a buffer so it does not need those annotations, and it does write `Data` into `"value"`. The json representation always uses `"value"`.

Tests that cover the actual inputs and outputs, like the exchange tests, are useful to detect these [de]serialization errors that passed the more focused unit tests.